src/apns.erl

```diff
@@ -40,16 +41,14 @@
              , response/0
              , token/0
              , headers/0
-             , stream_id/0
              ]).
 
--type json()      :: #{binary() | atom() => binary() | json()}.
+-type json()      :: #{binary() => binary() | json()}.
```

Comment:
It's still looks stricter than allowed by `jsx` and also APNS allows "primitive types such as dictionary (object), array, string, number, and Boolean. "